include/drivers/mspi/nrfe_mspi.h

```diff
@@ -30,47 +30,62 @@ extern "C" {
 #endif
 
 /** @brief eMSPI opcodes. */
-enum nrfe_mspi_opcode {
+typedef enum {
 	NRFE_MSPI_EP_BOUNDED = 0,
-	NRFE_MSPI_CONFIG_PINS,
-	NRFE_MSPI_CONFIG_CTRL, /* struct mspi_cfg */
-	NRFE_MSPI_CONFIG_DEV,  /* struct mspi_dev_cfg */
-	NRFE_MSPI_CONFIG_XFER, /* struct mspi_xfer */
-	NRFE_MSPI_TX,
+	NRFE_MSPI_CONFIG_PINS, /* nrfe_mspi_pinctrl_soc_pin_msg_t */
+	NRFE_MSPI_CONFIG_DEV,  /* nrfe_mspi_dev_config_msg_t */
+	NRFE_MSPI_CONFIG_XFER, /* nrfe_mspi_xfer_config_msg_t */
+	NRFE_MSPI_TX,	       /* nrfe_mspi_xfer_packet_msg_t + data buffer at the end */
 	NRFE_MSPI_TXRX,
 	NRFE_MSPI_WRONG_OPCODE,
 	NRFE_MSPI_ALL_OPCODES = NRFE_MSPI_WRONG_OPCODE,
-};
+} nrfe_mspi_opcode_t;
```

Comment:
Is this expected to be upstreamed? if so, please don't define new types. Is there a specific reason for making this a type?

Reply:
No, this is strictly for SDP so downstream.